drydock/templates/kustomized/tutor15/extensions/drydock-jobs/lms.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: lms-job
+  labels:
+    app.kubernetes.io/component: job
+    drydock.io/target-service: lms
+    drydock.io/runner-service: lms
+  annotations:
+    argocd.argoproj.io/sync-wave: "2"
+    argocd.argoproj.io/hook: Sync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  ttlSecondsAfterFinished: 100
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: lms
+        image: {{ DOCKER_IMAGE_OPENEDX }}
+        command: ["/bin/sh", "-c"]
+        args:
+        - |
+          dockerize -wait tcp://{{ MYSQL_HOST }}:{{ MYSQL_PORT }} -timeout 20s
+          {%- if MONGODB_HOST.startswith("mongodb+srv://") %}
+          echo "MongoDB is using SRV records, so we cannot wait for it to be ready"
+          {%- else %}
+          dockerize -wait tcp://{{ MONGODB_HOST }}:{{ MONGODB_PORT }} -timeout 20s
+          {%- endif %}
+
+          echo "Loading settings $DJANGO_SETTINGS_MODULE"
+
+          ./manage.py lms migrate
+
+          # Create oauth2 apps for CMS SSO
+          # https://github.com/openedx/edx-platform/blob/master/docs/guides/studio_oauth.rst
+          ./manage.py lms manage_user {{ DRYDOCK_CMS_SSO_USER }} {{ DRYDOCK_CMS_SSO_USER }}@openedx --unusable-password
+          ./manage.py lms create_dot_application \
+            --grant-type authorization-code \
+            --redirect-uris "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ CMS_HOST }}/complete/edx-oauth2/" \
+            --client-id {{ CMS_OAUTH2_KEY_SSO }} \
+            --client-secret {{ CMS_OAUTH2_SECRET }} \
+            --scopes user_id \
+            --skip-authorization \
+            --update cms-sso cms-sso-user
```

Comment:
We added the variable `DRYDOCK_CMS_SSO_USER` because we once found an existing user with the username `cms` and we couldn't use that one for authentication. The problem here is that  `cms-sso-user` is hard coded and instead should be `{{ DRYDOCK_CMS_SSO_USER }}`.